utils/uploads_used.py

```diff
@@ -80,3 +84,43 @@ def query_uploads_used(plan_service, limit, owner):
         )
 
     return queryset[:limit].count()
+
+
+def query_monthly_coverage_measurements(plan_service: PlanService) -> int:
+    owner = plan_service.current_org
+    queryset = UserMeasurement.objects.filter(
+        owner=owner,
+        private_repo=True,
+        created_at__gte=timezone.now() - timedelta(days=30),
+        report_type="coverage",
+    )
+    if (
+        plan_service.trial_status == TrialStatus.EXPIRED.value
+        and plan_service.has_trial_dates
+    ):
+        queryset = queryset.filter(
+            Q(created_at__gte=plan_service.trial_end_date)
+            | Q(created_at__lte=plan_service.trial_start_date)
+        )
```

Comment:
Do we need to filter out `ONGOING` rows?

Reply:
When you're ongoing, you're in the `users-trial` plan, so there's no monthly_limit associated with it, so it automatically takes care of this